docs/writing-tests/rendering.md

```diff
@@ -80,7 +76,7 @@ ways:
  * Silver or light gray is often used for irrelevant content, such as
    filler text.
 
-Obviously, none of this can be taken as absolute rules, as testing of
+None of these rules are absolute because testing
```

Comment:
```suggestion
None of these rules are absolute, because testing
```

Reply:
This looks like a stylistic usage rather than one that clarifies meaning. I'd prefer to omit it unless you have a more objective rational.